Riot/Modules/Settings/Security/ManageSession/ManageSessionViewController.m

```diff
@@ -114,6 +114,8 @@ - (void)viewDidLoad
         
     }];
     [self userInterfaceThemeDidChange];
+    
+    [self registerDeviceChangesNotification];
```

Comment:
We could unregister at `viewDidDisapear`. But we can miss some changes if we comeback to the screen later and a change occurs in between. I would mean refresh at `viewWillAppear` every time.

Reply:
managed IRL. This is pointless here.